public/apps/customerror/custom-error.tsx

```diff
@@ -45,7 +45,7 @@ export function CustomErrorPage(props: CustomErrorDeps) {
         {props.subtitle}
       </EuiText>
       <EuiSpacer size="s" />
-      <EuiButton fill href={props.http.basePath.serverBasePath} fullWidth>
+      <EuiButton fill onClick={() => logout(props.http, '')} fullWidth>
         Back to OpenSearch Dashboards Home
```

Comment:
Do we want to change this text to "Logout"?

Reply:
+1, otherwise this change looks good to me

Reply:
Complete